SU2_CFD/src/solvers/CDiscAdjSolver.cpp

```diff
@@ -572,6 +572,7 @@ void CDiscAdjSolver::SetSurface_Sensitivity(CGeometry *geometry, CConfig *config
 
 void CDiscAdjSolver::Preprocessing(CGeometry *geometry, CSolver **solver_container, CConfig *config, unsigned short iMesh,
                                    unsigned short iRKStep, unsigned short RunTime_EqSystem, bool Output) {
+  config->SetGlobalParam(config->GetKind_Solver(), RunTime_EqSystem);
```

Comment:
Should be fine I think

Reply:
Awesome, then I'll start merging the PR's